ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/DirectionOrderMapper.java

```diff
@@ -55,6 +63,10 @@ public DirectionOrder from(CCDDirectionOrder ccdDirectionOrder, CCDOrderGenerati
             .extraDocUploadList(directionOrderData.getExtraDocUploadList().stream()
                 .map(CCDCollectionElement::getValue)
                 .collect(Collectors.toList()))
+            .expertReportInstructionsForClaimant(directionOrderData.getExpertReportInstructionClaimant()
+                .stream().map(CCDCollectionElement::getValue).collect(Collectors.toList()))
+            .expertReportInstructionsForDefendant(directionOrderData.getExpertReportInstructionDefendant()
+                .stream().map(CCDCollectionElement::getValue).collect(Collectors.toList()))
```

Comment:
Please add these in assertion and unit test.

Reply:
added